src/Components/SourcesSelect/index.js

```diff
@@ -20,7 +21,7 @@ const SourcesSelect = ({ setValidation }) => {
     error,
     isLoading,
     data: sources,
-  } = useQuery(SOURCES_QUERY_KEY, fetchSourcesList, {
+  } = useQuery(SOURCES_QUERY_KEY, fetchSourcesList(PROVIDER), {
```

Comment:
we need to pass a function, not it's result here
```suggestion
  } = useQuery(SOURCES_QUERY_KEY, () => fetchSourcesList(PROVIDER), {
```
Or another solution would be to change `fetchSourcesList` to return a function